client/src/pages/ReviewCreate.jsx

```
@@ -161,6 +160,7 @@ export default function ReviewCreate() {
               <ImageUploader
                 originalFileName={originalFileName}
                 image={image}
+                imageUrl={imageUrl}
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)